packages/react-devtools-extensions/src/utils.js

```diff
@@ -1,10 +1,35 @@
 /* global chrome */
 
-import type {BrowserTheme} from 'react-devtools-shared/src/devtools/views/DevTools';
+import type { BrowserTheme } from 'react-devtools-shared/src/devtools/views/DevTools';
 
-export const IS_EDGE = navigator.userAgent.indexOf('Edg') >= 0;
-export const IS_FIREFOX = navigator.userAgent.indexOf('Firefox') >= 0;
-export const IS_CHROME = IS_EDGE === false && IS_FIREFOX === false;
+let IS_EDGE;
+let IS_FIREFOX;
+let IS_CHROME;
+
+async function getBrowserData() {
+  if (navigator.userAgentData) {
+    const userAgentData = await navigator.userAgentData.getHighEntropyValues([
```

Comment:
Can you get `brands` like this?

```suggestion
   const brands = navigator.userAgentData.brands;
```

Reply:
navigator.userAgentData.brands doesn't work because userAgentData is a method, not an object, and it returns a promise. That's why we have to use it as navigator.userAgentData.getHighEntropyValues(['brands']). This method returns a promise that resolves with an object containing the requested user agent data.

Reply:
> navigator.userAgentData.brands doesn't work because userAgentData is a method, not an object, and it returns a promise. That's why we have to use it as navigator.userAgentData.getHighEntropyValues(['brands']). This method returns a promise that resolves with an object containing the requested user agent data.

Yes, `getHighEntropyValues` returns Promise, but I don't understand why we need this method, using just `navigator.userAgentData.brands` works for me:
<img width="860" alt="Screenshot 2023-06-21 at 15 36 56" src="https://github.com/facebook/react/assets/28902667/b104ab33-fe51-4118-a591-f2c4e2ee65c3">

Reply:
> 
While it's true that navigator.userAgentData is an object that includes the brands property, I think it is recommended to use getHighEntropyValues() for more granular and specific information.

However, as navigator.userAgentData.brands works, i can certainly use that instead. Thanks for the review :)